apps/desktop/src/app/login/login.component.scss

```diff
@@ -0,0 +1,24 @@
+@import 'variables';
+.splash {
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  z-index: 99999;
```

Comment:
is this super high z-index really needed?

Reply:
that's the value we use for splash screen in our different index.html (that's why there is no SCSS variable for it, but I'll create one)